utils/yoke/derive/src/lib.rs

```diff
@@ -13,14 +13,19 @@ use synstructure::Structure;
 
 /// Custom derive for `yoke::Yokeable`,
 ///
-/// If this fails to compile for lifetime issues, it means that
-/// the lifetime is not covariant and `Yokeable` is not safe to implement.
+/// If your struct contains `zerovec::ZeroMap`, then the compiler will not
+/// be able to guarantee the lifetime covariance due to the generic types on
+/// the `ZeroMap` itself. You must add the following attribute in order for
+/// the custom derive to work with `ZeroMap`.
 ///
-/// Note that right now this will fail to compile on structs involving
-/// `zerovec::ZeroMap`.
-/// Please comment on <https://github.com/unicode-org/icu4x/issues/844>
-/// if you need this
-#[proc_macro_derive(Yokeable)]
+/// ```rust,ignore
+/// #[derive(Yokeable)]
+/// #[yoke(manually_prove_covariance)]
```

Comment:
Nit: This is the wrong spelling (manually_prove_covariance vs prove_covariance_manually)